### DISCUSSION TITLE
vcpkg need nasm in ubuntu

### DISCUSSION THREAD

**Discussion by zenjan1** [rustdesk/rustdesk#4479] [Show and tell]
When use vcpkg install ,

# vcpkg/vcpkg install libvpx libyuv opus aom
Computing installation plan...
The following packages will be built and installed:
    aom[core]:x64-linux -> 3.5.0#1
  * libjpeg-turbo[core]:x64-linux -> 2.1.5.1#1
    libvpx[core]:x64-linux -> 1.12.0#1
    libyuv[core]:x64-linux -> 1857
    opus[core]:x64-linux -> 1.4
  * vcpkg-cmake-get-vars[core]:x64-linux -> 2023-03-02
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-linux...
Restored 0 package(s) from /root/.cache/vcpkg/archives in 2.29 ms. Use --debug to see more details.
Installing 1/6 aom:x64-linux...
Building aom[core]:x64-linux...
CMake Error at scripts/cmake/vcpkg_find_acquire_program.cmake:163 (message):
  Could not find nasm.  Please install it via your package manager:

      sudo apt-get install nasm
Call Stack (most recent call first):
  ports/aom/portfile.cmake:2 (vcpkg_find_acquire_program)
  scripts/ports.cmake:147 (include)

error: building aom:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+aom
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?title=[aom]+Build+error&body=Copy+issue+body+from+%2Froot%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md

Just get nasm install

**Comment by 21pages** [comment]
sudo apt-get install nasm
